documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -62,7 +62,7 @@ KERNEL=kernel
 make bcmrpi_defconfig
 ----
 
-For Raspberry Pi 2, Pi 3, Pi 3+, and Compute Module 3 default build configuration
+For Raspberry PI Zero 2 W, Pi 2, Pi 3, Pi 3+, and Compute Module 3 default build configuration
```

Comment:
@pelwell Should "Compute Module 3+" also be listed here and on line 189?

Reply:
CM3 can be considered to include CM3, but feel free to add it - and don't ask for a review.

Reply:
Tidied up in #2428